torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -1350,6 +1350,7 @@ class ShapePropagator {
         {
           "aten::cumprod(Tensor self, int dim, *, int? dtype) -> Tensor",
           "aten::cumsum(Tensor self, int dim, *, int? dtype) -> Tensor",
+          "aten::cummax(Tensor self, int dim, *, int? dtype) -> (Tensor, Tensor)",
```

Comment:
look at the "Requirements above", particularly: 
``` //   scalar type    : dtype if specified, preserved if floating point,
    //    otherwise long/int64	    //    otherwise long/int64```

I don't think that behavior is sensible for cummax, so it should definitely not be in this list and we should just get rid of the parameter IMO.

Reply:
oh yeah sorry didn't realize. what do you mean by the "parameter"? are you talking about the second return value(indices)?

Reply:
yeah didn't realize. thanks for pointing it out